packages/azure_frontdoor/changelog.yml

```
@@ -0,0 +1,6 @@
+# newer versions go on top
+- version: "0.0.1"
```

```suggestion
- version: "0.1.0"
```
Since system tests are added, this can be considered a beta of sorts and hence the version bump.